files/jvm/src/main/java/com/swoval/runtime/NativeLoader.java

```diff
@@ -106,7 +106,8 @@ private static void loadPackaged(final String arch) throws IOException {
                     System.getProperty(
                         "swoval.tmpdir", System.getProperty("java.io.tmpdir", "/tmp")))
                 .resolve("swoval-jni"));
-    final String resourcePath = "/native/" + arch + "/" + lib;
+    final String osSubDir = System.getProperty("os.name").startsWith("FreeBSD") ? "/freebsd/" : "/";
```

Comment:
Even though I'd imagine it's pretty safe to assume that `os.name` is set, I'd prefer if this was `System.getProperty("os.name", "")`

Reply:
I agree, I'll fix.